includes/content-gate/class-content-gate.php

```diff
@@ -167,6 +167,11 @@ public static function do_blocks( $content ) {
 	 * Register post type for custom gate.
 	 */
 	public static function register_post_type() {
+		$icon = sprintf(
+			'data:image/svg+xml;base64,%s',
+			base64_encode( \Newspack\Newspack_UI_Icons::get_svg( 'key' ) )
```

Comment:
NIT: same here, no prefix needed as the classes share the same namespace.

```suggestion
			base64_encode( Newspack_UI_Icons::get_svg( 'key' ) )
```